spec/controllers/organizations_controller_spec.rb

```diff
@@ -5,6 +5,7 @@
 
   let(:organization)  { GitHubFactory.create_owner_classroom_org }
   let(:user)          { organization.users.first                 }
+  let(:student)       { GitHubFactory.create_classroom_student   }
```

Comment:
@johndbritton but that's not supposed to be the role of this user. The student is someone who isn't apart of the organization.